packaging/msi/create_windows_msi.py

```diff
@@ -96,6 +97,8 @@ def build(dist_path: Path, filename: Path):
     link_command = ["light", f"{build_loc.joinpath(wxs_loc.name).with_suffix('.wixobj')}",
                     f"{build_loc.joinpath(heat_loc.name).with_suffix('.wixobj')}",
                     f"{build_loc.joinpath(manageoldcuradlg_loc.name).with_suffix('.wixobj')}",
+                    "-sw1076",
+                    "-dcl:high",
```

Comment:
Can we add a comment what these arguments do, and why they are needed?

Reply:
Will do